### ISSUE TITLE
Pay-As-You-Go Calling Plan really required for shared calling?

### ISSUE THREAD

**Issue by JMCA2** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#10432]
On this page and on https://learn.microsoft.com/en-us/microsoftteams/shared-calling-setup and https://learn.microsoft.com/en-us/powershell/module/teams/new-csteamssharedcallingroutingpolicy?view=teams-ps it mentions assigning a pay-as-you-go calling plan.

Is it really required to use the kind of plan? It's not possible to assign a standard domestic calling plan with minutes included?
I understand for large organizations those minutes included might be insufficient when shared, but for small orgs it might be enough and it should be clarified on the docs if it really is necessary to opt for pay as you go.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: dae4f3dc-c617-d8b8-8a6b-bda47a6e9755
* Version Independent ID: dae4f3dc-c617-d8b8-8a6b-bda47a6e9755
* Content: [Plan for Shared Calling - Microsoft Teams](https://learn.microsoft.com/en-us/microsoftteams/shared-calling-plan)
* Content Source: [Teams/shared-calling-plan.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/public/Teams/shared-calling-plan.md)
* Service: **msteams**
* GitHub Login: @CarolynRowe
* Microsoft Alias: **crowe**

**Comment by scanum** [comment]
Thank you for submitting feedback.
I think the best way forward is if you open a service ticket in your tenant so this can get resolved ASAP. Based on the outcome let me know if it is something that can be called out in the docs.
Please follow this link to contact support for business products:https://docs.microsoft.com/office365/admin/contact-support-for-business-products
Please keep us posted here on the resolution so we can feed whatever information you discover into the content.

**Comment by ArtemioPantoja** [comment]
Hello @JMCA2 
Did you get answer to your question?
I'm trying to configure Shared Calling, with a Domestic Calling plan, and It is not working for me.

thanks

**Comment by JMCA2** [comment]
@ArtemioPantoja We contacted Microsoft Support and after several days no response

**Comment by JMCA2** [comment]
@ArtemioPantoja  @scanum We received response from Microsoft support and they say only pay-as-you-go plan is supported, but their response is solely based on reading the docs...

**Comment by ArtemioPantoja** [comment]
@JMCA2, thanks for keeping me posted.
From my side, I opened a case with ms, they reviewed some logs we provided and they found out an error regarding funds to our account communications credits.

"phrase": "Error from CreateUsageRequest:\r\nBalance not found"

We will try funding those comm credits and see what happens. I'll let you know if I have any luck.

Btw, we are using a Domestic calling plan for the resource account.
